python/cog/director/director.py

```diff
@@ -262,7 +258,7 @@ def _handle_message(self, message_id: str, message_json: str) -> None:
                     tracker.update_from_webhook_payload(event.payload)
                 elif isinstance(event, HealthcheckStatus):
                     log.info("received healthcheck status update", data=event)
-                    if event.health != Health.HEALTHY:
+                    if event.health not in {Health.BUSY, Health.READY}:
```

Comment:
Do we want to handle the model container saying it's `READY` without having sent a terminal webhook? My instinct is we worry about then if and when we observe it being a problem.

Reply:
I think there is a race condition here: we might get the READY healthcheck update before the webhook event, so I think we have to allow this here.

Reply:
Yeah, there's definitely a race condition here. It'd have to be some more complicated logic, like if it flips to ready we should receive the terminal webhook within 5s otherwise we'd abort it, similar to the cancelation bit. Which feels like a lot of work for something that may not even be a problem in practice!